### PR TITLE
Fix video bleeding in information blocks

### PR DESCRIPTION
**Pull request by xswoodsx**
## Ticket and context

Ticket: [755](https://dfedigital.atlassian.net/jira/software/projects/CPDEL/boards/89?assignee=5f3543bb3e9e2e004db3189d&selectedIssue=CPDEL-755)

### Is there anything weird that code reviewer should know?
Nope, everything is straightforward.

### Review Checks
- [ ] All commit messages are meaningful and true
- [ ] Added enough automated tests
- [ ] All forms have an `id` attribute on them

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Go to `/teach-first/year-1/autumn-1/topic-4/part-2` for videos in an information block that have been resized. 
3. Go to `/ambition/year-1/autumn-1/topic-2/part-1` or find a video not in an information block to check it's not had an effect on other videos. 

**Information block youtube video**
Screenshot 2021-08-02 at 08 41 33

**Details spacing**
Screenshot 2021-08-02 at 08 43 26

### CODE REVIEW

File: app/webpacker/styles/govspeak/_highlights.scss [hunk 1]
```diff
@@ -4,5 +4,9 @@
     border-radius: 9px;
     padding: 40px;
     margin: 50px;
+    iframe {
+      width: 444px;
+      height: 269px;
+    }
```
Comment (kicferk1) [inline app/webpacker/styles/govspeak/_highlights.scss:10]: Does that override the default values we set in govspeak? So the videos in information blocks are smaller than the regular ones?

File: app/webpacker/styles/govspeak/_details.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.gem-c-govspeak {
+  .govuk-details {
+    padding-top: 5px;
+  }
+}
```
Comment (kicferk1) [inline app/webpacker/styles/govspeak/_details.scss:5]: What does this do? (Or rather, why is it here? :D )

Comment (xswoodsx) [reply app/webpacker/styles/govspeak/_details.scss:5]: It just adds a tiny bit of space after a video for the details section. It's just to make it look a bit nicer. I'll add an image.

Comment (xswoodsx) [reply app/webpacker/styles/govspeak/_details.scss:5]: I mentioned it to Dom and she just said if have somewhere to add it in then add a bit of spacing

File: app/webpacker/styles/govspeak/_highlights.scss (hunk 1 above)
Comment (xswoodsx) [reply app/webpacker/styles/govspeak/_highlights.scss:10]: It doesn't look too. If you go to `/ambition/year-1/autumn-1/topic-2/part-1` the video is the size set in govspeak - the old size as is the case in this review app

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**